geometry_msgs/Pose pose
bool async
---
bool result
